src/plots/cartesian/layout_attributes.js

```diff
@@ -478,6 +478,20 @@ module.exports = {
             'to the left/bottom of labels.'
         ].join(' ')
     },
+    ticklabelmode: {
+        valType: 'enumerated',
+        values: ['', 'period'],
+        dflt: '',
```

Comment:
Per https://github.com/plotly/plotly.js/issues/4911 can we make the default be `'instant'` instead of `''`?

Reply:
We could consider it for sure.
However, I wanted to avoid testing strings here https://github.com/plotly/plotly.js/blob/4292f08339470f00e6266698f842bd30fb009145/src/plots/cartesian/axes.js#L2535 which may slow down a bit.

Reply:
Yes, let's use `'instant'`. That particular test isn't in a very hot path, and I don't think an exact string match is all that much overhead anyway.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4993/commits/9f3e1db2b3475e54db5492a57fc087efb3797fb4.